### DISCUSSION TITLE
trigger a large number of dagruns

### DISCUSSION THREAD

**Discussion by qinyang666** [apache/airflow#14716] [Q&A]
I have 3 DAG's and I try to trigger each DAG 3000 times.I get a bunch of exceptions. Here is my custom operator and the error I encountered. My version of Airlfow is 2.0.1

![image](https://user-images.githubusercontent.com/22069999/110762034-5733a700-828b-11eb-87af-20d8d28c8508.png)

![image](https://user-images.githubusercontent.com/22069999/110762146-73374880-828b-11eb-9589-f652d9db3e92.png)

![image](https://user-images.githubusercontent.com/22069999/110762394-b72a4d80-828b-11eb-8b11-50deba3dd6be.png)

**Comment by turbaszek** [comment]
This seems like resource limitation. What is your setup? What you are trying to achieve is to run 9k DAGs at the same moment which asks for big deployment...

**Reply by qinyang666** [reply]
scheduler 8C * 3
worker 8C * 5 
worker_concurrency ：16
parallelism 1000
max_active_runs_per_dag 1000
dag_concurrency 1000

**Reply by turbaszek** [reply]
So you have maximum of 16 * 5 = 80 tasks running at the same time. I would consider increasing worker concurrency or increasing the timeouts.
